sycl/test/sub_group/barrier.cpp

```diff
@@ -44,16 +44,10 @@ template <typename T> void check(queue &Queue, size_t G = 240, size_t L = 60) {
           sgsizeacc[0] = SG.get_max_local_range()[0];
       });
     });
-
-    // Temporary workaround to avoid waiting for the task while still blocked
-    // task is located in the same queue.
-    size_t sg_size = 0;
-    {
-      auto sgsizeacc = sgsizebuf.get_access<access::mode::read_write>();
-      sg_size = sgsizeacc[0];
-    }
-
     auto addacc = addbuf.template get_access<access::mode::read_write>();
+    auto sgsizeacc = sgsizebuf.get_access<access::mode::read_write>();
```

Comment:
Please, add some information about the fix.
What we did to fix the problem?
If it's the commit from this PR, please, consider merging them together or reference the commit id.

Reply:
I just revert changes I made earlier to work around problems that now resolved in  "[SYCL] Use exclusive queues for blocked commands. "

Reply:
I think it should be part of the "[SYCL] Use exclusive queues for blocked commands. " commit then.

Reply:
Ok, merged.